unicodetools/data/ucd/dev/EastAsianWidth.txt

```diff
@@ -2624,6 +2624,7 @@ FFFD           ; A  # So         REPLACEMENT CHARACTER
 1F6D0..1F6D2   ; W  # So     [3] PLACE OF WORSHIP..SHOPPING TROLLEY
 1F6D3..1F6D4   ; N  # So     [2] STUPA..PAGODA
 1F6D5..1F6D7   ; W  # So     [3] HINDU TEMPLE..ELEVATOR
+1F6D8          ; N  # So         LANDSLIDE
```

Comment:
Do we know why some of these are W vs. N?

Reply:
Oh. They should probably be W, like all recent emoji. https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp?a=%5Cp%7BEmoji%7D&g=age+ea&i=

Reply:
Interesting... especially weird that the "User interface symbols" from Unicode 6 are inconsistent with each other.

Reply:
At the risk of summoning an historian to find out why, U+231A and U+231B were changed from eaw=N to eaw=W in Unicode 9.0.